frontend/src/components/data-table/columns.tsx

```diff
@@ -217,6 +220,72 @@ export function generateColumns<T>({
           !isCellSelected;
 
         const format = column.getColumnFormatting?.();
+
+        if (typeof value === "string" && value.length > MAX_STRING_LENGTH) {
+          if (format) {
+            const formattedValue = column.applyColumnFormatting(value);
+            return (
+              <PopoutColumn
```

Comment:
could this be refactored a bit to:

const getValue() = {
  // handles formatting
  // handle renderValue
}

to that we don't need to use `<PopoutColumn` twice